### PR TITLE
article page display

### PR DESCRIPTION
**Pull request by g-kimani**

### CODE REVIEW

File: src/components/ArticlePage.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+              <FavoriteBorderOutlinedIcon />
+              {article.votes}
+            </Button>
+          </span>
+        </p>
+      </div>
+      <div>
+        <p>{article.body}</p>
+      </div>
+      <ArticleComments article_id={article_id} />
```
Comment (MKD01) [inline src/components/ArticlePage.jsx]: This ticket should not include comments

**Review by MKD01 — CHANGES_REQUESTED** [review]
This pull request has two tickets combined into one. Double check your trello board tickets and use this as a guide for each pull request. 

Please split these two tickets up and send them through as separate pr requests.

File: src/components/ArticlePage.jsx (hunk 1 above)
Comment (g-kimani) [reply src/components/ArticlePage.jsx]: Okay no problem

**Review by Keb84 — APPROVED** [review]
Great work for ticket 5. Ready to merge 🙂

**Pull request merged**